### ISSUE TITLE
Intermittent BMP/Soil Type D incompatibility

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/model-my-watershed#1037]
```
Task 8d9e8df4-f52b-4a7b-81ac-21902b575384 run from job 7856 raised exception: BMP rain_garden incompatible with soil d
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "apps/modeling/tasks.py", line 194, in run_tr55
    model_output = simulate_day(aoi_census, precip, cell_res=resolution)
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 395, in simulate_day
    return simulate_modifications(census, fn, cell_res, precolumbian)
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 344, in simulate_modifications
    simulate_water_quality(mod, cell_res, fn, precolumbian=precolumbian)
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 278, in simulate_water_quality
    cell, precolumbian)
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 278, in simulate_water_quality
    cell, precolumbian)
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 298, in simulate_water_quality
    result = fn('%s:%s:%s' % tuple(split), n)  # runoff, et, inf
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 393, in fn
    return simulate_cell_day(precip, et, cell, cell_count)
  File "/usr/local/lib/python2.7/dist-packages/tr55/model.py", line 136, in simulate_cell_day
    inf = lookup_bmp_infiltration(soil_type, bmp)
  File "/usr/local/lib/python2.7/dist-packages/tr55/tablelookup.py", line 42, in lookup_bmp_infiltration
    raise Exception('BMP %s incompatible with soil %s' % (bmp, soil_type))
```

**Comment by SRGDamia1** [comment]
I think this has been fixed with the changes made to the BMP implementation.

**Comment by SRGDamia1** [comment]
I think this has been fixed with the new way the BMP's are being implemented.

**Comment by SRGDamia1** [comment]
![screenshot](https://cloud.githubusercontent.com/assets/5374350/13268701/f9bd2d9e-da51-11e5-94a4-15e67ea768ad.jpg)

**Comment by mmcfarland** [comment]
Fixed with release from #1065
